packages/data-store/src/entities/contact/ContactMetadataItemEntity.ts

```diff
@@ -0,0 +1,50 @@
+import { Entity, Column, PrimaryGeneratedColumn, BaseEntity, ManyToOne, BeforeInsert, BeforeUpdate } from 'typeorm'
+import { ValidationConstraint } from '../../types'
+import { BaseContactEntity } from './BaseContactEntity'
+import { IsNotEmpty, validate, ValidationError } from 'class-validator'
+import { getConstraint } from '../../utils/ValidatorUtils'
+
+@Entity('ContactMetadata')
+export class ContactMetadataItemEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string
+
+  @Column({ name: 'label', length: 255, nullable: false })
+  @IsNotEmpty({ message: 'Blank metadata labels are not allowed' })
+  label!: string
+
+  @Column({ name: 'valueType', nullable: false })
+  @IsNotEmpty({ message: 'valueType must not be empty' })
+  valueType!: string
+
+  @Column({ name: 'stringValue', length: 255, nullable: true })
+  stringValue?: string
+
+  @Column({ name: 'numberValue', nullable: false })
```

Comment:
it says nullable false, but the field is optional? same for all other fields and identity

Reply:
Fixed
Apparently this attribute doesn't do anything atm. The migrations file did not contain these errors so, tests like "should get a party by contact metadata" didn't fail.